docs/source/AdministratorGuide/Systems/DataManagement/index.rst

```diff
@@ -43,92 +43,125 @@ DIRAC provides an abstraction of a SE interface that allows to access different
+        }
+      }
+    }
+    StorageElements
+    {
+      CERN-DST-EOS
+      {
+        BaseSE = CERN-EOS
+      }
+      CERN-USER
+      {
+        BaseSE = CERN-EOS
+        PledgedSpace = 205
+        AccessProtocol.1
+        {
+          PluginName = GFAL2_SRM2
+          Path = /eos/lhcb/grid/user
+          SpaceToken = LHCb_USER
+        }
+      }
+      GFAL2_XROOT
       {
-        Host = srm-eoslhcb.cern.ch
+        Host = eoslhcb.cern.ch
         Port = 8443
-        PluginName = GFAL2_SRM2
-        Protocol = srm
-        Path = /eos/lhcb/grid/prod
+        Protocol = root
+        Path = /eos/lhcb/grid/user
         Access = remote
         SpaceToken = LHCb-EOS
         WSUrl = /srm/v2/server?SFN=
       }
-   }
-   CERN-USER
-   {
-      BackendType = Eos
-      SEType = T0D1
-      PledgedSpace = 205
-      AccessProtocol.1
+    }
+
+
+This definition would be strictly equivalent to::
+
+    StorageElementBases
+    {
+      CERN-EOS
+      {
+        BackendType = Eos
+        SEType = T0D1
+        AccessProtocol.1
+        {
+          Host = srm-eoslhcb.cern.ch
+          Port = 8443
+          PluginName = GFAL2_SRM2
+          Protocol = srm
+          Path = /eos/lhcb/grid/prod
+          Access = remote
+          SpaceToken = LHCb-EOS
+          WSUrl = /srm/v2/server?SFN=
+        }
+      }
+    }
+    StorageElements
+    {
+      CERN-DST-EOS
       {
-        Host = srm-eoslhcb.cern.ch
+        BackendType = Eos
+        SEType = T0D1
+        AccessProtocol.1
+        {
+          Host = srm-eoslhcb.cern.ch
+          Port = 8443
+          PluginName = GFAL2_SRM2
+          Protocol = srm
+          Path = /eos/lhcb/grid/prod
+          Access = remote
+          SpaceToken = LHCb-EOS
+          WSUrl = /srm/v2/server?SFN=
+        }
+      }
+      CERN-USER
+      {
+        BackendType = Eos
+        SEType = T0D1
+        PledgedSpace = 205
+        AccessProtocol.1
+        {
+          Host = srm-eoslhcb.cern.ch
+          Port = 8443
+          PluginName = GFAL2_SRM2
+          Protocol = srm
+          Path = /eos/lhcb/grid/user
+          Access = remote
+          SpaceToken = LHCb_USER
+          WSUrl = /srm/v2/server?SFN=
+        }
+      }
+      GFAL2_XROOT
+      {
+        Host = eoslhcb.cern.ch
         Port = 8443
-        PluginName = GFAL2_SRM2
-        Protocol = srm
+        PluginName =  GFAL2_XROOT
+        Protocol = root
         Path = /eos/lhcb/grid/user
         Access = remote
-        SpaceToken = LHCb_USER
+        SpaceToken = LHCb-EOS
         WSUrl = /srm/v2/server?SFN=
       }
-   }
+    }
 
-Note that in case of overwriting, the protocol sections are matched using the PluginName attribute, and not the section name, which is irrelevant at all.
+Note that base SE must be separated from the inherited SE in two different sections. You can also notice that the name of the protocol section can be a plugin name. In this way, you do not need to specify a plugin name inside.
```

Comment:
Don't we want to make that a rule at some point?
The current implementation eases the migration, but at some point we should not have the PluginName any longer, right?